src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -101,9 +101,10 @@
     "ng2-charts": "2.4.2",
     "ngx-pipe-function": "^1.0.0",
     "ngx-toastr": "13.1.0",
+    "path": "0.12.7",
```

Comment:
Do we really need this?

Reply:
This was needed with swagger-ui-dist. Removed now. Thanks